Jenkinsfile

```diff
@@ -33,7 +33,7 @@ pipeline {
                     "-Dengine.page.url=${params.engineSource} " +
                     "-Divy.engine.version.latest.minor=true " +
                     "-Dskip.gpg=false " +
-                    "-Dgpg.passphrase='${env.GPG_PWD}' " +
+                    "-Dgpg.passphraseEnvName=GPG_PWD " +
```

Comment:
really enhances the security: the environment variable 'GPG_PWD' is already set by the brace `withCredentials` so there is no need to forward the plain text pass along.
now we only tell maven from which env variable the password must bread.